string name
int32 blockSize
---
float32[] deltaBlock
int32[] boundries
int32[] rasterizedRoom